lib/utils/cookie.js

```diff
@@ -22,30 +25,32 @@ cookie.prototype.get = function(str){
   var data = {};
 
   if (Cookies.get(this.config.key)) {
-    data = json.parse( decodeURIComponent(Cookies.get(this.config.key)) );
+    data = json.parse(Cookies.get(this.config.key));
   }
   if (str) {
-    return ('undefined' !== typeof data[str]) ? data[str] : null;
+    return (typeof data[str] !== 'undefined') ? data[str] : null;
   }
   else {
     return data;
   }
 };
 
-cookie.prototype.set = function(str, value){
-  if (!arguments.length || !this.enabled()) return this;
-  if ('string' === typeof str && arguments.length === 2) {
+cookie.prototype.set = function(str, value, daysUntilExpire){
+  if (!arguments.length || !this.enabled) return this;
+  if (typeof str === 'string'  && arguments.length === 2) {
     this.data[str] = value ? value : null;
   }
-  else if ('object' === typeof str && arguments.length === 1) {
+  else if (typeof str === 'object' && arguments.length === 1) {
```

Comment:
How would I set an `object` with an expiration? Looks like maybe this case is excluded here?